README.md

```diff
@@ -53,6 +53,34 @@ describe("BananService") {
 
 Each `it` block should have only one assertion. This is in line with the _Isolated_ principle.
 
+```swift
+describe("Movie") {
+  context("when initializing from dictionary") {
+  
+    let name = "The Dark Knight Rises"
+    let genre = "Action"
+    let year = 2012
+    let movie: Movie!
+    
+    beforeEach {
+      movie = Movie(dict: ["name": name,
+                           "genre": genre,
+                           "year": year])
+    }
+
+    it("maps name property") {
+      expect(movie.name) == name
+    }
+    it("maps genre property") {
+      expect(movie.genre) == genre
+    }
+    it("maps year property") {
+      epxect(movie.year) == year
+    }
```

Comment:
The only thing that I'd say is that because "tests should read like sentences" I'd add a _the_ in the `it` blocks string.

```
describe("Movie") {
  context("when initializing from dictionary") {
    it("maps **the** name property") { ... }
    ...
  }
}

// => "Movie, when initializing from dictionary, it maps the name property"
```